app/extensions/brave/locales/en-US/extensions.properties

```diff
@@ -29,3 +29,5 @@ torrent= Torrent Viewer
 torrentDesc=Uses WebTorrent to display torrents directly in the browser. Supports torrent files and magnet links.
 vimium= Vimium
 vimiumDesc=
+honey=Honey
```

Comment:
@kevinlawler We should probably open up a PR to merge your `cookies.getAll` stub into Muon. Thoughts? cc @bsclifton